native/src/main/java/io/ballerina/persist/cmd/Generate.java

```diff
@@ -0,0 +1,252 @@
+        SyntaxTree configTree = BalSyntaxTreeGenerator.generateConfigBalFile();
+        try {
+            writeOutputFile(configTree, Paths.get(this.sourcePath, "modules",
+                            "generated_clients", "database_configuration.bal")
+                    .toAbsolutePath().toString());
+        } catch (IOException e) {
+            throw new GenerateBalException("Error occurred in formatting database configuration file!");
+        } catch (FormatterException e) {
+            throw new GenerateBalException("Error occurred while generating database configuration file!");
+        }
+    }
+
+    private static void writeOutputFile(SyntaxTree syntaxTree, String outPath) throws IOException, FormatterException {
+        String content;
+        content = Formatter.format(syntaxTree.toSourceCode());
+        Path pathToFile = Paths.get(outPath);
+        Files.createDirectories(pathToFile.getParent());
+        try (PrintWriter writer = new PrintWriter(outPath, StandardCharsets.UTF_8.name())) {
+            writer.println(content);
+        }
+    }
+    public static boolean hasSyntacticDiagnostics(Path filePath) {
+        String content;
+        try {
+            content = Files.readString(filePath);
+        } catch (IOException e) {
+            return false;
+        }
+        TextDocument textDocument = TextDocuments.from(content);
+        return SyntaxTree.from(textDocument).hasDiagnostics();
+    }
+
+    public static boolean hasSemanticDiagnostics(Path projectPath,
+                                                 ProjectEnvironmentBuilder projectEnvironmentBuilder) {
+        Package currentPackage;
+        if (projectEnvironmentBuilder == null) {
+            BuildProject buildProject = BuildProject.load(projectPath.toAbsolutePath());
+            currentPackage = buildProject.currentPackage();
+        } else {
+            BuildProject buildProject = BuildProject.load(projectEnvironmentBuilder, projectPath);
+            currentPackage = buildProject.currentPackage();
+        }
+        PackageCompilation compilation = currentPackage.getCompilation();
+        DiagnosticResult diagnosticResult = compilation.diagnosticResult();
+        return diagnosticResult.hasErrors();
+    }
+
+    public void setSourcePath(String sourcePath) {
+        this.sourcePath = sourcePath;
+    }
+    public void setEnvironmentBuilder(ProjectEnvironmentBuilder projectEnvironmentBuilder) {
+        this.projectEnvironmentBuilder = projectEnvironmentBuilder;
+    }
+
+    @Override
+    public void setParentCmdParser(CommandLine parentCmdParser) {
+        this.parentCmdParser = parentCmdParser;
+    }
+    @Override
+    public String getName() {
+        return PersistToolsConstants.COMPONENT_IDENTIFIER;
+    }
+    
+    @Override
+    public void printLongDesc(StringBuilder out) {
+        out.append("Generated the Client object").append(System.lineSeparator());
```

Comment:
```suggestion
        out.append("Generate Client objects corresponding to persist entities").append(System.lineSeparator());
```

Reply:
updated in [6d8b4a4](https://github.com/ballerina-platform/persist-tools/pull/15/commits/6d8b4a41b9a0ea1002bb906bb2008380cfa7cdb1)